Makefile

```diff
@@ -25,58 +31,58 @@ bin: # bin creates the binaries for Waypoint for the current platform
 # You can use the binary it produces as a server, runner, or CLI, but it won't contain the CEB, so
 # it won't be able to build projects that don't have `disable_entrypoint = true` set in their build hcl.
 .PHONY: bin/no-ceb
-bin/cli-only:
+bin/cli-only: # Builds only the cli with no ceb
 	CGO_ENABLED=$(CGO_ENABLED) go build -ldflags $(GOLDFLAGS) -tags assetsembedded -o ./waypoint ./cmd/waypoint
 
 .PHONY: bin/linux
-bin/linux: # bin creates the binaries for Waypoint for the linux platform
+bin/linux: # Creates the binaries for Waypoint for the linux platform
 	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o ./internal/assets/ceb/ceb ./cmd/waypoint-entrypoint
 	CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -o ./internal/assets/ceb/ceb-arm64 ./cmd/waypoint-entrypoint
 	cd internal/assets && go-bindata -pkg assets -o prod.go -tags assetsembedded ./ceb
 	GOOS=linux CGO_ENABLED=$(CGO_ENABLED) go build -ldflags $(GOLDFLAGS) -tags assetsembedded -o ./waypoint ./cmd/waypoint
 
 .PHONY: bin/windows
-bin/windows: # create windows binaries
+bin/windows: # Create windows binaries
 	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o ./internal/assets/ceb/ceb ./cmd/waypoint-entrypoint
 	CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -o ./internal/assets/ceb/ceb-arm64 ./cmd/waypoint-entrypoint
 	cd internal/assets && go-bindata -pkg assets -o prod.go -tags assetsembedded ./ceb
 	GOOS=windows GOARCH=amd64 CGO_ENABLED=$(CGO_ENABLED) go build -ldflags $(GOLDFLAGS) -tags assetsembedded -o ./waypoint.exe ./cmd/waypoint
 
 .PHONY: bin/entrypoint
-bin/entrypoint: # create the entrypoint for the current platform
+bin/entrypoint: # Create the entrypoint for the current platform
 	CGO_ENABLED=0 go build -tags assetsembedded -o ./waypoint-entrypoint ./cmd/waypoint-entrypoint
 
 .PHONY: install
-install: bin # build and copy binaries to $GOPATH/bin/waypoint
+install: bin # Build and copy binaries to $GOPATH/bin/waypoint
 ifneq ("$(wildcard $(GOPATH)/bin/waypoint)","")
 	rm $(GOPATH)/bin/waypoint
 endif
 	mkdir -p $(GOPATH)/bin
 	cp ./waypoint $(GOPATH)/bin/waypoint
 
 .PHONY: format
-format: # format go code
+format: # Format all go code in project
 	gofmt -s -w ./
 
 .PHONY: docker/server
 docker/server: docker/server-only docker/odr
```

Comment:
Do we care about these targets not formatting like the rest of the targets?

Reply:
e.g. When looking at `docker/gen/server` it also calls `docker/tools` and the output looks like this as a header.
```
docker/gen/server: docker/tools
```

We can exclude them by modifying your regex above to be
```
^[a-zA-Z/_-]+:\s+#' Makefile | sort | awk 'BEGIN {FS = ":.*?# "}; {printf "\033[36m%-15s\033[0m %s\n", $$1, $$2}
```

Reply:
I think it's ok since they're like a "meta" target that runs other targets. Maybe in a follow-up PR if someone is motivated?

Reply:
Works for me

Reply:
I'll steal your regex!